### PR TITLE
Milestone 2 : Create Work Section and About Section

### PR DESCRIPTION
**Pull request by belalgamal46**
Milestone 2 Work Section and About section:
- Create section HTML tag
- Implemented Grid and adjust my rows and columns
- Create a card section
- Use the card section created in all other work cards and changed the title and image
- Create about section html
- Style about section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -66,6 +257,41 @@ <h1>I’m Belal Gamal Glad to see you!</h1>
...
+          </div>
+          <div class="skills-list-style">
+            <h4>Frameworks</h4>
+            <img src="./images//right-arrow.svg" alt="arrow" />
+          </div>
+          <hr />
+          <div class="skills-list-style">
+            <h4>Skills</h4>
+            <img src="./images//right-arrow.svg" alt="arrow" />
+          </div>
```
Comment (MaganaAsiati) [inline index.html:292]: - Here you did a great job, your design looks very nice but makes sure this section is visible in the mobile version. as I show you in the picture below I'm only able to see the frameworks but not the skills. kindly ensure that you fix this issue :grinning:  :grinning:  :grinning: 
## Your design 
 
image

## Figma design
 
image

**Review by MaganaAsiati — CHANGES_REQUESTED** [review]
# STATUS: Required Changes ♻️♻️♻️ 
## Hi @belalgamal46 , I'm @MaganaAsiati, your reviewer for this PR  :wave: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-gif-4

## Highlights

- Descriptive README file  :heavy_check_mark:👨🏻‍💻
- Great work using the right GitHub Flow.:heavy_check_mark:
- No Linter errors :heavy_check_mark:

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._
- [ ] [OPTIONAL] Please make sure to make your commit messages more descriptive in the future. :heavy_check_mark:👨🏻‍💻


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by belalgamal46** [comment]
@MaganaAsiati at my laptop it is visible
Screenshot from 2022-09-23 17-05-39

File: index.html (hunk 1 above)
Comment (belalgamal46) [reply index.html:292]: @MaganaAsiati it is visible on my laptop
Screenshot from 2022-09-23 17-05-39

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @belalgamal46 , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**